src/storage/kura.rs

```diff
@@ -170,43 +173,122 @@ impl MerkleTree {
     }
 }
 
+static DEFAULT_BLOCK_STORE_LOCATION: &str = "./blocks/";
```

Comment:
It's better to hardcode this parameter for now and remove `'a`.

Looks quite dirty right now - we have a lot of unnecessary lifetime specifiers to use a string in one place.

Reply:
If I remove `block_store_location` field, the logic with cleaning temporary up folders will also have to be removed. Not sure if it's a good idea.

Reply:
Maybe we can make it into an owned `String` and owned `PathBuf` to remove the lifetime specifiers. What do you think @humb1t ?

Reply:
@eadventurous let's start from owned solution